src/edit/main.js

```diff
@@ -339,7 +350,7 @@ class ProseMirror {
   flush() {
     this.unscheduleFlush()
 
-    if (!document.body.contains(this.wrapper) || !this.operation) return false
+    if (!this.root.contains(this.wrapper) || !this.operation) return false
```

Comment:
It looks like the intention of `document.body.contains(this.wrapper)` was basically "is the editor in the DOM?". This means this code can probably be executed even if the editor *isn't* in the DOM. You probably want this condition to be `!this.root || !this.root.contains(this.wrapper) || !this.operation`.